pass-journal-loader/pass-journal-loader-nih/src/main/java/org/eclipse/pass/loader/journal/nih/LoaderEngine.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright 2017 Johns Hopkins University
```

Comment:
Do you want to update the Copyright year to 2023?

Reply:
That's a good question for the group. I would only usually only update the copyright year if the something was substantively rewritten, but I don't think we have any policy. I'll add a topic to our next sprint meeting.